types/mercadopago/models/payment/capturePartial-payload.model.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface CapturePartialPayload {
+    /** Payment id. */
+    id: number;
+
+    /** New amount. */
+    transaction_amount: number;
```

Comment:
Are you sure the description is /** New amount. */? I have tested and transaction_amount is optional, so make it opitional. Also, if set transaction_amount should be the exact value as the one when you created, otherwise we would get an error. So I really think that we should not include this property as it could add confusion.

Reply:
According to MERCADOPAGO, you can capture a value less than the reserved, you only get an error when trying to capture a value higher than the reserved. EX: You reserve $20 + $10 by delivery fee, but the tax is reduced to $5, so you must update and capture the amount.

Reply:
It does make sense and I have tryed this but it does not work. Can you provide any example? Because I have tested this and it does not work. I also could not find any information about this on the API Reference (but we all know that their api reference is totally incomplete). When I test this capture thing with a transaction_amount being less than the original I get the error message `Invalid parameters for payment_method API`

Reply:
How are you create payment? You need to pass `capture` parameter as false.

Try to create like this (change data in uppercase to your data):
```
curl -X POST \
    -H 'accept: application/json' \
    -H 'content-type: application/json' \
    -H 'Authorization: Bearer YOUR_ACCESS_TOKEN' \
    'https://api.mercadopago.com/v1/payments' \
    -d '{
	"token": "YOUR_CARD_TOKEN",
	"installments":1,
	"transaction_amount":58.80,
	"description":"Test",
	"payment_method_id": "MASTER_or_VISA",
	"issuer_id": "CARD_ISSUER_ID",
	"capture": false,
	"payer":{
		"id": "USER_ID",
		"email":"USER_EMAIL",
		"identification": {
                	"number": "NUMBER",
                	"type": "CNPJ_OR_CPF"
            	}
	},
	"sponsor_id":null,
	"binary_mode":false,
	"external_reference":"TESTE123",
	"statement_descriptor":"TEST APP",
	"additional_info":{
		"items":[
		{
			"id":"PR0001",
			"title":"Point Mini",
			"description": "Producto Point para cobros con tarjetas mediante bluetooth",
			"picture_url":"https://http2.mlstatic.com/resources/frontend/statics/growth-sellers-landings/device-mlb-point-i_medium@2x.png",
			"category_id": "electronics",
			"quantity":1,
			"unit_price":58.80
		}
		],
		"payer":{
			"first_name":"Nome",
			"last_name":"Sobrenome",
			"address":{
				"zip_code":"06233-200",
				"street_name":"Av das Nacoes Unidas",
				"street_number":3003
			},
			"registration_date":"2019-01-01T12:01:01.000-03:00",
			"phone":{
				"area_code":"011",
				"number":"987654321"
			}
		},
		"shipments":{
			"receiver_address":{
				"street_name":"Av das Nacoes Unidas",
				"street_number":3003,
				"zip_code":"06233200",
				"city_name": "Buzios",
				"state_name": "Rio de Janeiro"
			}
		}
  	}
}'
```

Reply:
I already did this whole thing. I even get the response with `"status_detail": "pending_capture"` but it still does not work, the error response is:
```
{
    "message": "Invalid parameters for payment_method API",
    "error": "bad_request",
    "status": 400,
    "cause": [
        {
            "code": 1004,
            "description": "Invalid parameters for payment_method API",
            "data": null
        }
    ]
}
```
As I said before, it only works if I set the transaction_amount to the save value as the created. I have tested everything. Also the example you gave does not make that much sense. Think of Uber, that simply cancel the payment and create a new one. Also MercadoPago has the `payments/:paymentId/refunds` endpoint so you can refund totally or partially. I am asking for changes because this transaction_amount should not be mandatory. Please remove this property so I can accept the PR.

Reply:
I changed the parameter to optional. I really don't know why this error is returning to you, I think it's the famous "it works on my machine". Regarding usage, this feature is currently useful to me because my app is for grocery stores, so when one item in the order is missing, instead of canceling the payment and making another, we simply capture the value by discounting the missing item. It's really hard work with this API, their reference is totally abandoned.